web/packages/teleport/package.json

```diff
@@ -28,6 +28,8 @@
     "@gravitational/build": "^1.0.0",
     "@types/wicg-file-system-access": "^2020.9.5",
     "jest-canvas-mock": "^2.3.1",
-    "ts-loader": "^9.4.2"
+    "ts-loader": "^9.4.2",
+    "babel-plugin-transform-import-meta": "^2.2.0",
```

Comment:
Not sure if these deps were added manually or automatically, but if manually I find it is best to keep lists like this in alphabetical order. It makes it easy to know when you have duplicates (or more accurately you don't get duplicates because it is obvious) and it helps manage merge conflicts when things get added at different places rather than all at the end. If these were added automatically by some command, please ignore this comment.

Reply:
a80054fdfaaec1b3ad8eb44e632baca3b2173a2f